### DISCUSSION TITLE
Stand up Tuesday May 16th

### DISCUSSION THREAD

**Discussion by BrandonMagana** [Bugs-io/TC3001B#78] [Main Stand Ups]
Share your daily standup update with the team here. Respond to this
discussion before the standup starts in the following format:

```
**What did you achieve in the last 24 hours?**:
- 

**What are your priorities for the next 24 hours?**:
- 

**Blockers**:
- 

**Shoutouts**:
- @username for x
```

**Comment by CarlosCRG19** [comment]
**What did you achieve in the last 24 hours?**:
- Nothing to be honest

**What are your priorities for the next 24 hours?**:
- Help @BrandonMagana on the deploy
- Work on graphics and (maybe) on endpoint to retrieve data

**Blockers**:
- Lizza stuff

**Shoutouts**:
- @everyone for attending class yesterday

**Comment by CristianCazares** [comment]
**What did you achieve in the last 24 hours?**:
- Merge PR's and solved a couple fixes
- Integrated the deployed api to vercel
- Created https://github.com/Bugs-io/ibm-dashboard-api/issues/20 

**What are your priorities for the next 24 hours?**:
- Start working on the grid

**Comment by BrandonMagana** [comment]
**What did you achieve in the last 24 hours?**:
-  Almost done automating deployment to gcloud run

**What are your priorities for the next 24 hours?**:
-  Finish the deployment github action

**Blockers**:
-  Don't know why it is failing 

**Shoutouts**:
- @CarlosCRG19 for participating in Adan's class

**Comment by Jacobgtz024** [comment]
**What did you achieve in the last 24 hours?**:
- Finish plotting some charts
- Generate questions to answer in the dashboard

**What are your priorities for the next 24 hours?**:
- Answer the questions with graphs
- Share the data in endpoints

**Blockers**:
- Incomplete data

**Shoutouts**:
- @todos
